src/components/mode-tools/mode-tools.jsx

```diff
@@ -192,7 +191,6 @@ const ModeToolsComponent = props => {
                 </InputGroup>
                 <InputGroup className={classNames(styles.modLabeledIconHeight)}>
                     <LabeledIconButton
-                        disabled={!props.selectedItems.length}
```

Comment:
I was looking into this and it's more work than I thought, because we aren't really keeping track in the state right now of anything that could indicate that the canvas is empty. I think we should file it as a separate issue as you suggested at first.

Reply:
Filed https://github.com/LLK/scratch-paint/issues/653